scripts.js

```diff
@@ -1,19 +1,13 @@
 function add_doggo() {
-    fetch("https://dog.ceo/api/breed/shiba/images")
-        .then(function(response) {
-            return response.json();
-        })
-        .then(function(imageObj) {
-            show_image(imageObj.message[0], 500,500, "Doggo");
-        });
+    fetch("https://dog.ceo/api/breed/shiba/images/random")
+        .then(response => response.json())
+        .then(show_image);
+}
+
+
+function show_image(dogObj) {
```

Comment:
One other trick here, you could use destructuring to pull out the message from the dogObj:
```
function show_image({message}) {
  const current_image = document.getElementById("static-doggo");
  current_image.src = message
```